src/grid.ts

```diff
@@ -0,0 +1,146 @@
+﻿/// <reference path="../scripts/typings/jquery/jquery.d.ts" />
```

Comment:
If you include a file in the Visual Studio solution, there is no need to add this reference